<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="A2ML29037" ignored="0" name="Default test" tests="6" failures="6" timestamp="27 Sep 2018 09:57:01 GMT" time="0.72" errors="0">
  <testcase name="Test_01" time="0.635" classname="ApiRestAutomation.RestApi.GetUserdetails">
    <failure type="java.lang.NoClassDefFoundError" message="Lgroovy/lang/Tuple2;">
      <![CDATA[java.lang.NoClassDefFoundError: Lgroovy/lang/Tuple2;
at java.lang.Class.getDeclaredFields0(Native Method)
at java.lang.Class.privateGetDeclaredFields(Unknown Source)
at java.lang.Class.getDeclaredFields(Unknown Source)
at org.codehaus.groovy.reflection.CachedClass$1$1.run(CachedClass.java:48)
at java.security.AccessController.doPrivileged(Native Method)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:45)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:43)
at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:46)
at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:33)
at org.codehaus.groovy.reflection.CachedClass.getFields(CachedClass.java:254)
at groovy.lang.MetaClassImpl.addFields(MetaClassImpl.java:2363)
at groovy.lang.MetaClassImpl.inheritFields(MetaClassImpl.java:2358)
at groovy.lang.MetaClassImpl.setupProperties(MetaClassImpl.java:2248)
at groovy.lang.MetaClassImpl.addProperties(MetaClassImpl.java:3255)
at groovy.lang.MetaClassImpl.initialize(MetaClassImpl.java:3226)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:210)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:241)
at com.jayway.restassured.internal.ResponseSpecificationImpl.$getStaticMetaClass(ResponseSpecificationImpl.groovy)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:49)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:64)
at com.jayway.restassured.RestAssured.createTestSpecification(RestAssured.java:1447)
at com.jayway.restassured.RestAssured.when(RestAssured.java:761)
at ApiRestAutomation.RestApi.GetUserdetails.Test_01(GetUserdetails.java:15)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: groovy.lang.Tuple2
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 48 more
]]>
    </failure>
  </testcase> <!-- Test_01 -->
  <testcase name="Test_02" time="0.005" classname="ApiRestAutomation.RestApi.GetUserdetails">
    <failure type="java.lang.NoClassDefFoundError" message="Lgroovy/lang/Tuple2;">
      <![CDATA[java.lang.NoClassDefFoundError: Lgroovy/lang/Tuple2;
at java.lang.Class.getDeclaredFields0(Native Method)
at java.lang.Class.privateGetDeclaredFields(Unknown Source)
at java.lang.Class.getDeclaredFields(Unknown Source)
at org.codehaus.groovy.reflection.CachedClass$1$1.run(CachedClass.java:48)
at java.security.AccessController.doPrivileged(Native Method)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:45)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:43)
at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:46)
at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:33)
at org.codehaus.groovy.reflection.CachedClass.getFields(CachedClass.java:254)
at groovy.lang.MetaClassImpl.addFields(MetaClassImpl.java:2363)
at groovy.lang.MetaClassImpl.inheritFields(MetaClassImpl.java:2358)
at groovy.lang.MetaClassImpl.setupProperties(MetaClassImpl.java:2248)
at groovy.lang.MetaClassImpl.addProperties(MetaClassImpl.java:3255)
at groovy.lang.MetaClassImpl.initialize(MetaClassImpl.java:3226)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:210)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:241)
at com.jayway.restassured.internal.ResponseSpecificationImpl.$getStaticMetaClass(ResponseSpecificationImpl.groovy)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:49)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:64)
at com.jayway.restassured.RestAssured.createTestSpecification(RestAssured.java:1447)
at com.jayway.restassured.RestAssured.when(RestAssured.java:761)
at ApiRestAutomation.RestApi.GetUserdetails.Test_02(GetUserdetails.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: groovy.lang.Tuple2
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 48 more
]]>
    </failure>
  </testcase> <!-- Test_02 -->
  <testcase name="Test_03" time="0.004" classname="ApiRestAutomation.RestApi.GetUserdetails">
    <failure type="java.lang.NoClassDefFoundError" message="Lgroovy/lang/Tuple2;">
      <![CDATA[java.lang.NoClassDefFoundError: Lgroovy/lang/Tuple2;
at java.lang.Class.getDeclaredFields0(Native Method)
at java.lang.Class.privateGetDeclaredFields(Unknown Source)
at java.lang.Class.getDeclaredFields(Unknown Source)
at org.codehaus.groovy.reflection.CachedClass$1$1.run(CachedClass.java:48)
at java.security.AccessController.doPrivileged(Native Method)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:45)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:43)
at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:46)
at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:33)
at org.codehaus.groovy.reflection.CachedClass.getFields(CachedClass.java:254)
at groovy.lang.MetaClassImpl.addFields(MetaClassImpl.java:2363)
at groovy.lang.MetaClassImpl.inheritFields(MetaClassImpl.java:2358)
at groovy.lang.MetaClassImpl.setupProperties(MetaClassImpl.java:2248)
at groovy.lang.MetaClassImpl.addProperties(MetaClassImpl.java:3255)
at groovy.lang.MetaClassImpl.initialize(MetaClassImpl.java:3226)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:210)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:241)
at com.jayway.restassured.internal.ResponseSpecificationImpl.$getStaticMetaClass(ResponseSpecificationImpl.groovy)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:49)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:64)
at com.jayway.restassured.RestAssured.createTestSpecification(RestAssured.java:1447)
at com.jayway.restassured.RestAssured.given(RestAssured.java:741)
at ApiRestAutomation.RestApi.GetUserdetails.Test_03(GetUserdetails.java:33)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: groovy.lang.Tuple2
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 48 more
]]>
    </failure>
  </testcase> <!-- Test_03 -->
  <testcase name="Test_04" time="0.005" classname="ApiRestAutomation.RestApi.GetUserdetails">
    <failure type="java.lang.NoClassDefFoundError" message="Lgroovy/lang/Tuple2;">
      <![CDATA[java.lang.NoClassDefFoundError: Lgroovy/lang/Tuple2;
at java.lang.Class.getDeclaredFields0(Native Method)
at java.lang.Class.privateGetDeclaredFields(Unknown Source)
at java.lang.Class.getDeclaredFields(Unknown Source)
at org.codehaus.groovy.reflection.CachedClass$1$1.run(CachedClass.java:48)
at java.security.AccessController.doPrivileged(Native Method)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:45)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:43)
at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:46)
at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:33)
at org.codehaus.groovy.reflection.CachedClass.getFields(CachedClass.java:254)
at groovy.lang.MetaClassImpl.addFields(MetaClassImpl.java:2363)
at groovy.lang.MetaClassImpl.inheritFields(MetaClassImpl.java:2358)
at groovy.lang.MetaClassImpl.setupProperties(MetaClassImpl.java:2248)
at groovy.lang.MetaClassImpl.addProperties(MetaClassImpl.java:3255)
at groovy.lang.MetaClassImpl.initialize(MetaClassImpl.java:3226)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:210)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:241)
at com.jayway.restassured.internal.ResponseSpecificationImpl.$getStaticMetaClass(ResponseSpecificationImpl.groovy)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:49)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:64)
at com.jayway.restassured.RestAssured.createTestSpecification(RestAssured.java:1447)
at com.jayway.restassured.RestAssured.given(RestAssured.java:741)
at ApiRestAutomation.RestApi.GetUserdetails.Test_04(GetUserdetails.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: groovy.lang.Tuple2
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 48 more
]]>
    </failure>
  </testcase> <!-- Test_04 -->
  <testcase name="Test_05" time="0.004" classname="ApiRestAutomation.RestApi.GetUserdetails">
    <failure type="java.lang.NoClassDefFoundError" message="Lgroovy/lang/Tuple2;">
      <![CDATA[java.lang.NoClassDefFoundError: Lgroovy/lang/Tuple2;
at java.lang.Class.getDeclaredFields0(Native Method)
at java.lang.Class.privateGetDeclaredFields(Unknown Source)
at java.lang.Class.getDeclaredFields(Unknown Source)
at org.codehaus.groovy.reflection.CachedClass$1$1.run(CachedClass.java:48)
at java.security.AccessController.doPrivileged(Native Method)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:45)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:43)
at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:46)
at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:33)
at org.codehaus.groovy.reflection.CachedClass.getFields(CachedClass.java:254)
at groovy.lang.MetaClassImpl.addFields(MetaClassImpl.java:2363)
at groovy.lang.MetaClassImpl.inheritFields(MetaClassImpl.java:2358)
at groovy.lang.MetaClassImpl.setupProperties(MetaClassImpl.java:2248)
at groovy.lang.MetaClassImpl.addProperties(MetaClassImpl.java:3255)
at groovy.lang.MetaClassImpl.initialize(MetaClassImpl.java:3226)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:210)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:241)
at com.jayway.restassured.internal.ResponseSpecificationImpl.$getStaticMetaClass(ResponseSpecificationImpl.groovy)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:49)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:64)
at com.jayway.restassured.RestAssured.createTestSpecification(RestAssured.java:1447)
at com.jayway.restassured.RestAssured.given(RestAssured.java:741)
at ApiRestAutomation.RestApi.GetUserdetails.Test_05(GetUserdetails.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: groovy.lang.Tuple2
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 48 more
]]>
    </failure>
  </testcase> <!-- Test_05 -->
  <testcase name="Test_06" time="0.008" classname="ApiRestAutomation.RestApi.GetUserdetails">
    <failure type="java.lang.NoClassDefFoundError" message="Lgroovy/lang/Tuple2;">
      <![CDATA[java.lang.NoClassDefFoundError: Lgroovy/lang/Tuple2;
at java.lang.Class.getDeclaredFields0(Native Method)
at java.lang.Class.privateGetDeclaredFields(Unknown Source)
at java.lang.Class.getDeclaredFields(Unknown Source)
at org.codehaus.groovy.reflection.CachedClass$1$1.run(CachedClass.java:48)
at java.security.AccessController.doPrivileged(Native Method)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:45)
at org.codehaus.groovy.reflection.CachedClass$1.initValue(CachedClass.java:43)
at org.codehaus.groovy.util.LazyReference.getLocked(LazyReference.java:46)
at org.codehaus.groovy.util.LazyReference.get(LazyReference.java:33)
at org.codehaus.groovy.reflection.CachedClass.getFields(CachedClass.java:254)
at groovy.lang.MetaClassImpl.addFields(MetaClassImpl.java:2363)
at groovy.lang.MetaClassImpl.inheritFields(MetaClassImpl.java:2358)
at groovy.lang.MetaClassImpl.setupProperties(MetaClassImpl.java:2248)
at groovy.lang.MetaClassImpl.addProperties(MetaClassImpl.java:3255)
at groovy.lang.MetaClassImpl.initialize(MetaClassImpl.java:3226)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClassUnderLock(ClassInfo.java:210)
at org.codehaus.groovy.reflection.ClassInfo.getMetaClass(ClassInfo.java:241)
at com.jayway.restassured.internal.ResponseSpecificationImpl.$getStaticMetaClass(ResponseSpecificationImpl.groovy)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:49)
at com.jayway.restassured.internal.ResponseSpecificationImpl.<init>(ResponseSpecificationImpl.groovy:64)
at com.jayway.restassured.RestAssured.createTestSpecification(RestAssured.java:1447)
at com.jayway.restassured.RestAssured.given(RestAssured.java:741)
at ApiRestAutomation.RestApi.GetUserdetails.Test_06(GetUserdetails.java:86)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: groovy.lang.Tuple2
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 48 more
]]>
    </failure>
  </testcase> <!-- Test_06 -->
</testsuite> <!-- Default test -->
